source/jormungandr/tests/stats_tests.py

```diff
@@ -160,6 +162,39 @@ def check_stat_places_to_publish(self, stat):
         self.called = True
 
 
+class TestRabbitMqPublication(unittest.TestCase):
+    def _on_message(self, body, message):
+        self.has_message = True
+        message.ack()
+
+    def test_StatManager_can_publish(self):
+        self.has_message = False
+
+        app.config['SAVE_STAT'] = True
+        app.config['EXCHANGE_NAME'] = 'test_rabbitmq'
+
+        stat_mngr = StatManager()
+
+        queue = kombu.Queue(name="test_queue", exchange=stat_mngr.exchange, routing_key="bla")
+
+        bounded_queue = queue(stat_mngr.connection)
+        bounded_queue.declare()
+        bounded_queue.purge()
+
+        stat_mngr.publish_request("bla", "test")
+
+        with stat_mngr.connection.Consumer([bounded_queue], callbacks=[self._on_message]):
+            time_to_live = 100
+            while self.has_message == False and time_to_live > 0:
+                stat_mngr.connection.drain_events()
+                time_to_live -= 1
+
+        bounded_queue.delete()
```

Comment:
you can creating your queue by declaring it `auto_delete` so that you don't need to delete it manually